### ISSUE TITLE
Print a list of 

### ISSUE THREAD

**Issue by mark-russell-bossanova** [hadolint/hadolint#388]
- [ ] This is a bug report
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
Pass a command line flag to print all of the rules defined.  It would be nice to be able to get a list of all of the rules that are defined by passing a command line flag.

### Actual behavior
No flag exists

### Steps to reproduce the behavior
n/a

**Output of `hadolint --version` or
  `docker run --rm hadolint/hadolint hadolint --version`:**

```bash
Haskell Dockerfile Linter v1.17.5-2-g44d0caa
```

**Additional environment details (OS, stack version, etc.)**
I run on Ubuntu 16.4

**Comment by adriantorrie** [comment]
This would be good as the [list of rules](https://github.com/hadolint/hadolint#rules`) are incomplete (and noted as such).
